### PR TITLE
[docstring] Fix docstring for `ASTFeatureExtractor`

### PR DESCRIPTION
**Pull request by imsoumya18**
# What does this PR do?

Fixes #26638

## Who can review?

@ydshieh

### CODE REVIEW

**Comment by imsoumya18** [comment]
@ydshieh Why am I getting this error?
Screenshot from 2023-10-09 19-09-48

**Comment by abzdel** [comment]
Could just try `pip install tensorrt` - may not be completely necessary for your docstring change though.

Also, I would try `make fix-copies` in your next commit as it looks like this was what made CI fail

**Comment by imsoumya18** [comment]
Screenshot from 2023-10-09 21-48-29

**Comment by ydshieh** [comment]
You haven't done any fix but just remove one entry in the list to ignore. You should follow the issue page of this event.

**Comment by ydshieh** [comment]
Please run `python3 utils/check_docstrings.py --fix_and_overwrite` (or python)

**Comment by imsoumya18** [comment]
> Hi @imsoumya18 Do you encounter some issues when running `python3 utils/check_docstrings.py --fix_and_overwrite`?

Yes, I am continuously getting this error
image

**Comment by imsoumya18** [comment]
> I see, but I don't think it is an error, but just warning.
> 
> After you run that command, no file has being changed by the script? If so, I will take a look (but in this case, probably indeed an environment issue).

After it, no file was changed. But, I think (I maybe wrong) it's an error. Maybe a problem of the env. Also, I gave up hope and completely deleted the fork and repo whole. Let me fork and setup the env and check if the issue is solved. I will then open a new PR or reopen this one. Let me try again by setting up from start.

**Comment by ydshieh** [comment]
@imsoumya18 No worry, thank you for the effort. Environment is indeed frustrated, but maybe a new fresh python virtual environment is easy way. Let me know if you need any help. Look forward to your contribution 🔥
